### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by texttao** [SakuraRiven/EAST#2]
你好、我在运行一会之后就损失为nan了、请问能更改么

**Comment by SakuraRiven** [comment]
@texttao hi, can you share the train log? Generally, decreasing the learning rate may help.

**Comment by texttao** [comment]
It has not any useful information.But i see the result of "torch.sum(gt_score)" in "loss.py" is zero.

**Comment by SakuraRiven** [comment]
@texttao Thank you very much for your help! I would later fix this bug. Actually, "torch.sum(gt_score)" is zero because the gt has no text region. This rarely happens due to the batch_size is not small.

I think the loss could be set to 0 in this case. If this solution works, welcome to share the result : )
